packages/styled-icons/README.md

```diff
@@ -53,7 +56,36 @@ or
 npm install styled-icons --save
 ```
 
-Additionally, you will need to have installed a version of `styled-components` at least version 4.1.0 or newer, as `styled-icons` depends on `styled-components` as a peer dependency.
+Alternatively you can install only the icon packs you need:
+
+```
+yarn add @styled-icons/boxicons-logos
```

Comment:
I think we should think through how this would work - for some packs, every icon is available in every variant, but in others they're not.  Something like `import {AlertRounded} from '@styled-icons/pack'` could be cool, but I'm not sure all the implications of joining the packs into base icon packages.

Reply:
That would be interesting indeed. I guess what I was thinking is for a easier solution that aligns more with what we currently have.

For example boxions

```js
import { regular, solid, logos } from '@styled-icons/boxicons' 
import { Cookie } from '@styled-icons/boxicons/regular'
import { Cookie } from '@styled-icons/boxicons-regular'
```

For the case of entypo, maybe something like

```
import { regular, social } from '@styled-icons/entypo'
// Or import { Cookie, social } from '@styled-icons/entypo' 
import { Cookie } from '@styled-icons/entypo/regular'
import { Cookie } from '@styled-icons/entypo-regular'
```

And I would really like to see material -> material-filled